client/src/components/Unfollower.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import "./Unfollower.css";
+
+export default function Unfollower({ user }) {
+    console.log(user)
+
+    const userId = user.uri.split(':')[2];
+    // Render
+    if(!user.imageURL){
+        return (
+            <div className="unfollower" >
+                <a href={`https://open.spotify.com/user/${userId}`} target="_blank" rel="noreferrer">
+                    <img src="https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_1280.png" alt="Spotify user" />
```

Comment:
We can add a conditional to img source to prevent copying same div.

Reply:
Yep, you're right, I just took the easy way out in order to not waste the time.  I learnt the better solution from your branch 👍